src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/RazorDiagnosticsEndpoint.cs

```diff
@@ -69,6 +78,8 @@ public async Task<RazorDiagnosticsResponse> Handle(RazorDiagnosticsParams reques
                 throw new ArgumentNullException(nameof(request));
             }
 
+            _logger.LogInformation($"Received {request.Kind:G} diagnostic request for {request.RazorDocumentUri} with {request.Diagnostics.Length} diagnostics.");
```

Comment:
I'm not familiar with the `:G` syntax, what does that do?

Reply:
This is equivalent to `request.Kind.ToString("G")`.

The `G` [converts the enum _name_ to string](https://docs.microsoft.com/en-us/dotnet/standard/base-types/enumeration-format-strings#g-or-g) (`HTML` instead of writing out the value `2`).

Reply:
≈≈≈≈≈★ The More You Know.